### PR TITLE
test_5 added, lint errors fixed

### PR DESCRIPTION
**Pull request by TarasSkip**

### CODE REVIEW

File: pages/searchResults.page.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+let BasePage = require('./base.page');
+let TextBox = require('../elements/textBox');
+let WebButton = require('../elements/button');
+
+let searchResultsFirstLinkLocator = by.css('.products-list > li:nth-child(1) > div.item-info > p > a');
```
Comment (nbabii) [inline pages/searchResults.page.js]: please optimize locator

File: pages/searchResults.page.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    async waitForSearchPageToBeAvailable() {
+        await this.getPageHeaderElement().waitForElementToBeVisible(10000);
+    }
+
+    // action methods
+    async getSearchResults() {
+        await allure.createStep(`Getting search results (check that found '> 0' items)`, async () => {
+            let string = await this.getPageHeaderElement().getText();
+            let num = await string.replace(/[^0-9]/g, '');
+            let int = await parseInt(num, 10);
```
Comment (nbabii) [inline pages/searchResults.page.js]: no need `await ` in lines 17 and 18, there are no promises

File: pages/searchResults.page.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+let searchResultsFirstLinkLocator = by.css('.products-list > li:nth-child(1) > div.item-info > p > a');
+let itemPageHeaderLocator = by.css('div.heading > h1');
+
+class SearchResultsPage extends BasePage {
+    async waitForSearchPageToBeAvailable() {
+        await this.getPageHeaderElement().waitForElementToBeVisible(10000);
+    }
+
+    // action methods
+    async getSearchResults() {
```
Comment (nbabii) [inline pages/searchResults.page.js:14]: make this this method just to return the count of search results. No need to do any `expect` here, expectation should be only in test body.

File: specs/test_5.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+let MainPage = require("../pages/main.page");
+let SearchResultsPage = require("../pages/searchResults.page");
+
+describe('Hotline Protractor Test - Search suite', () => {
+    it('TC #5. Search results', async () => {
+        await MainPage.open();
+        await MainPage.waitForPageToBeAvailable();
+        await MainPage.search('samsung galaxy');
+        await SearchResultsPage.waitForSearchPageToBeAvailable();
+        await SearchResultsPage.getSearchResults();
```
Comment (nbabii) [inline specs/test_5.js]: Method `getSearchResults()` should be refactored

**Review by nbabii — CHANGES_REQUESTED** [review]

File: pages/searchResults.page.js (hunk 1 above)
Comment (TarasSkip) [reply pages/searchResults.page.js]: done

File: pages/searchResults.page.js (hunk 2 above)
Comment (TarasSkip) [reply pages/searchResults.page.js]: done

File: pages/searchResults.page.js (hunk 3 above)
Comment (TarasSkip) [reply pages/searchResults.page.js:14]: done

File: specs/test_5.js (hunk 4 above)
Comment (TarasSkip) [reply specs/test_5.js]: done

**Pull request merged**